modules/user/user.service.js

```diff
@@ -55,35 +78,37 @@ class UserService {
     });
 
     if (errors) {
-      throw new UserInputError('Errors', { errors });
+      throw new UserInputError('Errors', {
+        errors,
+      });
     }
 
     const user = await this.getUserByFieldOrThrow('_id', id);
-      
+
     if (user._doc.email !== email) {
       await this.checkUserExists(email);
     }
-    
+
     return User.findByIdAndUpdate(user._id, {
```

Comment:
Please, add third parameter ` {new: true}` to return already updated entity

Reply:
fixed